internal/c/libqb.cpp

```diff
@@ -13192,17 +13152,9 @@ void qbs_input(int32 numvariables, uint8 newline) {
                 c->chr[0] = 0;
                 qbs_set(qbs_input_arguements[argn], qbs_add(qbs_input_arguements[argn], c));
                 if (qbs_input_variabletypes[argn] & ISUNSIGNED) {
-#ifdef QB64_WINDOWS
-                    sscanf((char *)qbs_input_arguements[argn]->chr, "%I64u", (uint64 *)qbs_input_variableoffsets[argn]);
-#else
-                    sscanf((char *)qbs_input_arguements[argn]->chr, "%llu", (uint64 *)qbs_input_variableoffsets[argn]);
-#endif
+                    sscanf((char *)qbs_input_arguements[argn]->chr, "%" PRIu64, (uint64_t *)qbs_input_variableoffsets[argn]);
                 } else {

```

Comment:
Shouldn't the respective SCN... macros be used here (lines above and below) for scanf(), instead of the PRI... ones supposed for printf() just for clarity https://pubs.opengroup.org/onlinepubs/009695399/basedefs/inttypes.h.html

Reply:
Good catch.